content/docs/examples/tutorial/single/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Run a single microservice locally
+overview: Learn how to work on a single service on your local machine.
+weight: 10
+---
+
+{{< boilerplate work-in-progress >}}
+
+This module demonstrates different aspects of developing a single service on a local machine.
+We will not ask you to write code, your task will be to build, run and test an already implemented service,
+`ratings`.
+The goal is to show that you can implement, build, run and test a service on your local machine,
+before deploying it to a development, testing or production environment.
+
+The `ratings` service is a small web app written in [Node.js](https://nodejs.org/en/) that can run on its own.
+It performs the following actions, similar to the actions the web apps usually perform:
+
+- Listen to the port it receives as a parameter.
+- Expect `HTTP GET` requests on the `/ratings/{productID}` path and return the ratings of the product matching the value
+  you specify for `productID`.
+- Expect `HTTP POST` requests on the `/ratings/{productID}` path and update the ratings of the product matching the value
+  you specify for `productID`.
+
+Download the code of the app, install its dependencies and run it locally following the steps below:
+
+1.  Download
+    [the service's code]({{< github_blob >}}/samples/bookinfo/src/ratings/ratings.js)
+    and
+    [the package file]({{< github_blob >}}/samples/bookinfo/src/ratings/package.json)
+    into a separate directory:
+
+    {{< text bash >}}
+    $ mkdir ratings
+    $ cd ratings
+    $ curl -s {{< github_file >}}/samples/bookinfo/src/ratings/ratings.js -o ratings.js
+    $ curl -s {{< github_file >}}/samples/bookinfo/src/ratings/package.json -o package.json
+    {{< /text >}}
+
+1. Skim the service's code and note the following elements on the code:
+    - The web serving features
+        - listening to a port
+        - handling requests and responses
+    - The aspects related to HTTP:
+        - Headers
+        - Path
+        - Status code
+
+    A quick explanation for those who are not familiar with Node.js or some of other modern web application framework.
+    As opposed to the architecture of frameworks like [Apache Tomcat](http://tomcat.apache.org) or
+    [WebSphere Application Server](https://www.ibm.com/support/knowledgecenter/SS7K4U_9.0.5/com.ibm.websphere.zseries.doc/ae/welc6productov.html),
+    where one web server software runs multiple applications, in the case of Node.js the web serving functionality is
```

Comment:
@rcaballeromx Good idea to put it in a tip. I will remove "therefore", since "to run as a standalone process" does not necessarily follow from "being embedded in the code".

Reply:
Removed "Therefore" in https://github.com/istio/istio.io/pull/4704/commits/d647c5517a05f61917865dbb99452244302f6989

Reply:
@rcaballeromx For some reason "web serving" is not usually used in the context of web application frameworks. See, for example, documentation of Express, part of Node.js:
https://expressjs.com:

>  a robust set of features for web and mobile applications

> provides a thin layer of fundamental web application features

Reply:
@rcaballeromx I rewrote the tip in https://github.com/istio/istio.io/pull/4704/commits/322d0fd206f524bc97a3d7ab1a478f61033678fb